Library/Homebrew/cmd/doctor.rb

```diff
@@ -651,6 +651,14 @@ def check_user_curlrc
   end
 end
 
+def check_homebrew_bottle_domain
+  unless (bottle_domain = ENV["HOMEBREW_BOTTLE_DOMAIN"]).nil? then <<-EOS.undent
+    You have set HOMEBREW_BOTTLE_DOMAIN.
+    If you have trouble downloading bottles with Homebrew, then maybe this is the problem.
```

Comment:
`You have set HOMEBREW_BOTTLE_DOMAIN. This may cause issues when download formulae.`

Maybe it's enough for this to just be in `brew config` though?